app/uk/gov/hmrc/soletraderidentificationfrontend/testonly/stubs/controllers/BusinessVerificationStubController.scala

```diff
@@ -31,16 +31,26 @@ class BusinessVerificationStubController extends InjectedController {
 
   def createBusinessVerificationJourney: Action[JsValue] = Action.async(parse.json) {
     implicit request =>
-      val regime: String = (request.body \ "origin").as[String]
-      if (!regime.equals(regime.toLowerCase))
-        Future.failed(new IllegalArgumentException(s"origin value $regime has to be lower case, but it was not"))
-      else {
-        val continueUrl: String = (request.body \ "continueUrl").as[String]
-        Future.successful {
-          Created(Json.obj(
-            "redirectUri" -> (continueUrl + s"?journeyId=$businessVerificationJourneyId")
-          ))
-        }
+      val jsobBody = for {
```

Comment:
jsobBody → jsonBody
and if you agree we can remove the variables that are not used... up to you really

<img width="1049" alt="Screenshot 2022-04-28 at 10 48 59" src="https://user-images.githubusercontent.com/4958044/165726474-a05229c8-e600-4ef7-8fcf-2b8c10a27080.png">